flink-runtime-web/web-dashboard/app/scripts/modules/jobs/jobs.ctrl.coffee

```diff
@@ -318,3 +360,7 @@ angular.module('flinkApp')
   loadMetrics() if $scope.nodeid
 
 # --------------------------------------
+
+.controller 'JobPlanWatermarksController', ($scope) ->
```

Comment:
This controller is no longer empty, but we still needed to register it as a controller, so yes, we needed it.